sentry-core/src/test/java/io/sentry/core/transport/HttpTransportTest.kt

```diff
@@ -87,6 +90,8 @@ class HttpTransportTest {
         whenever(fixture.connection.inputStream).thenThrow(IOException())
         whenever(fixture.connection.getHeaderField(eq("Retry-After"))).thenReturn("30")
         whenever(fixture.connection.responseCode).thenReturn(429)
+        whenever(fixture.currentDateProvider.currentDate).thenReturn(Date())
```

Comment:
This returns the current date. So each run of the test is going to be with a different date. We could do `Date(0)` or `Date(2020, 12, 1, 8, 00, 00)` then this test would be deterministic.

Reply:
true, let me figure this out, don't forget the LOGAF please